src/screens/IndexScreen.js

```diff
@@ -209,27 +200,39 @@ export const IndexScreen = ({ navigation }) => {
   const fetchPolicy = graphqlFetchPolicy({ isConnected, isMainserverUp });
   const globalSettings = useContext(GlobalSettingsContext);
   const { filter = {} } = globalSettings;
-  const { news: showNewsFilter = false } = filter;
+  const { news: showNewsFilter = false, events: showEventsFilter = true } = filter;
```

Comment:
I don't get this code, what I get is  : 
filter is originally a empty object in globalSettings (line 202) 
filter can be news which is showNewsFilter = false or can be events  which is showEventsFilter = true .

Reply:
`globalSettings` comes from the server.
this will have a key `filter`. if not, `{}` is used as fallback.
in `filter` there should be `news` and `events`. if not, `false` is used as fallback for `news`, which will be renamed for further use to `showNewsFilter`. `true` is used as fallback for `events`, which will be renamed to `showEventsFilter`.

we use this javascript destructing of values in many places in our app. you need to get comfortable with it. maybe read about it in italian? https://developer.mozilla.org/it/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment